integration-tests/src/test/java/org/openmrs/maven/plugins/SetupIntegrationTest.java

```diff
@@ -164,6 +164,7 @@ public void setup_shouldInstallServerFromDistroPropertiesDir() throws Exception{
         assertFilePresent(serverId + File.separator + "frontend" + File.separator + "openmrs-esm-patient-chart-app-3.0.0");
         String indexContents = FileUtils.readFileToString(new File(testDirectory.getAbsolutePath(), indexFilePath));
         assertThat(indexContents, containsString("apiUrl: \"notopenmrs\""));
+        assertThat(indexContents, containsString("configUrls: [\"foo\"]"));
```

Comment:
It seems odd to me that we wouldn't test this with more than one array parameter, in order to test things like parsing the comma-separated syntax, etc.

Reply:
Yeah at present there's only one distro.properties file in the tests that includes the `spa.` properties, and the single array value seems more like an edge case which someone might overlook in implementing an array-values parser. Based on my experience having initially implemented this in a way that didn't support single array values :)

Could certainly add SPA building to another distro file but it would slow the tests down substantially.

Reply:
Maybe we could test the functionality of this in a unit test rather than adding a new integration test?

Reply:
That is extremely sensible

Reply:
Added some unit tests for `SpaInstaller`. Got halfway to adding unit test support for `Setup` too, until I realized that's not what we want here. But maybe that would be useful at some point.